src/eduid_webapp/signup/settings/common.py

```diff
@@ -68,4 +68,4 @@
 
 DEFAULT_FINISH_URL = 'https://www.eduid.se/'
 
-CURRENT_TOU_VERSION = 'test-version'
+INTERNAL_ACTIONS_URL = 'http://actions2.eduid.docker/'
```

Comment:
Please leave out the developer environment uri. If we don't have a sensible default it is better to have it as None as something will probably break if we forget to set it when deploying.

Reply:
Done